_deprecated/image_processing_app.yaml

```diff
@@ -26,13 +28,6 @@ spec:
           volumeMounts:
             - mountPath: "/mnt/mikro-testdata"
               name: test-volume
-          resources:
-            limits:
-              cpu: "1"
-              memory: 500Mi
-            requests:
-              cpu: 100m
-              memory: 200Mi
```

Comment:
i think we should just trust the namespace hard limit here, rather than faff about,

Reply:
That could work, but that way you'll have to always keep an eye at things and figure out what eats resources if things are crashing. But with the current workloads you're running this should not be an issue I guess